website/_data/compatibility.yml

```diff
@@ -3,9 +3,10 @@
 releases:
   - version: 0.18.0
     compatible:
-      - "2.30.0"
-      - "2.31.0"
+      - "2.33.0"
       - "2.32.0"
+      - "2.31.0"
+      - "2.30.0"
```

Comment:
Reordering happened because the previous update was manually applied. :shushing_face: